x-pack/plugin/searchable-snapshots/src/main/java/org/elasticsearch/xpack/searchablesnapshots/cache/CacheBufferedIndexInput.java

```diff
@@ -47,13 +48,13 @@
     // last seek position is kept around in order to detect forward/backward seeks for stats
     private long lastSeekPosition;
 
-    CacheBufferedIndexInput(CacheDirectory directory, FileInfo fileInfo, IOContext context, IndexInputStats stats) {
+    public CacheBufferedIndexInput(SearchableSnapshotDirectory directory, FileInfo fileInfo, IOContext context, IndexInputStats stats) {
```

Comment:
Has to be `public` now it's instantiated by `SearchableSnapshotDirectory` from a different package.